Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG FLOW_PHP_VERSION=8.2.11
+ARG FLOW_BASE_IMAGE_TAG_SUFFIX=cli-alpine3.18
+ARG FLOW_BASE_IMAGE_TAG=${FLOW_PHP_VERSION}-${FLOW_BASE_IMAGE_TAG_SUFFIX}
+ARG FLOW_BASE_IMAGE=php:${FLOW_BASE_IMAGE_TAG}
+
+FROM ${FLOW_BASE_IMAGE}
+
+ARG FLOW_VERSION=0.3.3
```

Comment:
This may need to be autogenerated maybe? Otherwise it will require manual change on release or when using it

Reply:
it's a default argument that should be provided otherwise, it will fallback to this version, gh actions workflow will take this from the event